libyul/YulStack.h

```diff
@@ -163,6 +166,8 @@ class YulStack: public langutil::CharStreamProvider
 	std::shared_ptr<yul::Object> m_parserResult;
 	langutil::ErrorList m_errors;
 	langutil::ErrorReporter m_errorReporter;
+
+	std::unique_ptr<std::string> m_sourceMappings;
```

Comment:
Ah, wait. Why did this come back? I remember removing it in one of my PRs because it was unused. Looks like rebasing error.